design/workitems.go

```diff
@@ -35,6 +35,7 @@ var workItem = a.Type("WorkItem", func() {
 var workItemRelationships = a.Type("WorkItemRelationships", func() {
 	a.Attribute("assignees", relationGenericList, "This defines assignees of the Work Item")
 	a.Attribute("labels", relationGenericList, "List of labels attached to the Work Item")
+	a.Attribute("system.boardcolumns", relationGenericList, "List of board columns this Work Item is attached to")
```

Comment:
why is this `system.boardcolumns`? It could be simply `boardcolumns`

Reply:
No, it should be `system.boardcolumns` because the field is named this way! I explicitly asked @michaelkleinhenz to call it this way. That is because soon all the other relationships will carry the name of their fields, so `labels` becomes `system.labels` and so forth. See also https://github.com/fabric8-services/fabric8-wit/issues/2028 and https://github.com/fabric8-services/fabric8-wit/issues/2029

Reply:
@DhritiShikhar actually you were right. To comply with JSONAPI we must not use "." in a relationship name. See https://github.com/fabric8-services/fabric8-wit/issues/2028#issuecomment-403189369